container-disc/src/main/sh/vespa-start-container-daemon.sh

```diff
@@ -90,6 +90,12 @@ configure_memory() {
     fi
 }
 
+configure_cpu() {
+    if ((jvm_availableProcessors > 0)); then
+        cpu_options="-XX:ActiveProcessorCount=${jvm_availableProcessors}"
+    fi
+}
```

Comment:
What will the value read from .getNumProcessors() if this is not set ?

Reply:
The thought was that since this is taken from a config and has default 0 it will always be set. Do you think otherwise? This is how it is done for jvm_heapSizeAsPercentageOfPhysicalMemory as well